app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ActiveRecord::Base
+  validates :name, presence: true, length: { maximum: 50 }
+  VALID_EMAIL_REGEXP = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  validates :email, presence: true, length: { maximum: 255 },
+              format: { with: VALID_EMAIL_REGEXP },
+              uniqueness: { case_sensitive: false }
```

Comment:
__[rubocop]__
* Align the elements of a hash literal if they span more than one line.